readthedocs/core/middleware.py

```diff
@@ -107,17 +107,17 @@ def process_request(self, request):
                         slug = cname_to_slug(host)
                         cache.set(host, slug, 60 * 60)
                         # Cache the slug -> host mapping permanently.
-                        log.debug(LOG_TEMPLATE.format(
+                        log.info(LOG_TEMPLATE.format(
```

Comment:
Is this one `info` for some particular reason?

Reply:
This only happens when caching the slug from a cname, doesn't feel like this should be a warning. The other logs are the _important part_